src/theme/foundations/z-index.ts

```diff
@@ -0,0 +1,16 @@
+export const zIndices = {
+  hide: -1,
+  auto: 'auto',
+  base: 0,
+  docked: 10,
+  dropdown: 1000,
+  sticky: 1100,
+  banner: 1200,
+  overlay: 1300,
+  modal: 1400,
+  popover: 1500,
+  skipLink: 1600,
+  toast: 1700,
+  select: 1750,
+  tooltip: 1800,
+};
```

Comment:
I would go for a lighter version that will be merge with Chakra default theme. If one day Chakra decides to change some zIndices, we won't be disturbed. Also, a comment at the beginning of the file is useful to quickly know where these values comes from.

```suggestion
// https://chakra-ui.com/docs/theming/theme#z-index-values
export const zIndices = {
  select: 1750,
};
```

Reply:
Done